task-list.jade

```diff
@@ -28,5 +28,4 @@
         |Completed
       label.btn.control(ng-click="filter('all')")
         input#All(type="radio", name="options") 
-        |All
-  //ng-click="edit(task.id)"
\ No newline at end of file
+        |All
```

Comment:
Github requires new lines at the EOF. Probably you can set this option in your IDE